crates/compiler/fmt/src/def.rs

```diff
@@ -117,7 +117,7 @@ impl<'a> Formattable for TypeDef<'a> {
                         buf,
                         Parens::NotNeeded,
                         Newlines::from_bool(make_multiline),
-                        indent + 1 + INDENT,
```

Comment:
It was introduced here

https://github.com/roc-lang/roc/pull/3546/files#diff-6b9bc54647cbdb26373a72f5e598e9dd4173a6b11c4264ed990e9386157d4e65R117

@ayazhafiz, was that intentional? The only actual thing I fixed in the pr is this line